components/hacbs/jvm-build-service/base/kustomization.yaml

```diff
@@ -4,6 +4,26 @@ resources:
 patchesStrategicMerge:
 - self-binding.yaml
 
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: IMAGE_TAG
```

Comment:
I intentionally did not institute the pre KCP flexibility we had with per image overrides in preview.env

Rationale: 
- even for the jvm bld svc dev flow, we only need to override the repo and tag to handle all the images 
- I'm still holding out hope I can switch back to the controller being able to construct the image refs for the cache and req processor (we no longer have sidecars or dep analyser usage) by the controller examining the image ref for its pod

Reply:
depending on how ^^ pans out, my thinking is we can always add those preview.env related override later on